bot/resources/fun/madlibs_templates.json

```diff
@@ -0,0 +1,135 @@
+             "value": ["Santa, Mrs. Claus, and the ", " live at the North pole. The weather is always ", " there, but the ", " ", " toys for Santa to ",
+                       " to children on Christmas, so holiday cheer lasts year-round there. There's no land at the North Pole; instead there is a ", "-inch thick sheet of ",
+                       " there, ", " enough to hold Santa's Village! The ", " help load Santa's sleigh with ", ", and Santa's ", " ", " his sleigh on Christmas Eve to ",
+                       " ", " to children around the entire world."]
+        },
+        {
+             "title": "Snowstorm!",
+             "blanks": ["plural noun", "adjective", "noun", "noun", "adjective", "adjective", "adjective", "noun", "noun", "adjective"],
+             "value": ["Weather plays an important part in our ", " everyday. What is weather, you ask? According to ", " scientists, who are known as meteorologists, weather is what the ",
+                       " is like at any given time of the ", ". It doesn't matter if the air is ", " or ", ", it's all weather. When vapors in ", " clouds condense, we have ",
+                       " and snow. A lot of ", " means a ", " snowstorm!"]
+        },
+        {
+             "title": "Learning About History",
+             "blanks": ["adjective", "noun", "nouns", "adjective", "nouns", "nouns", "animals", "nouns", "nouns", "number", "number", "nouns", "adjective", "nouns"],
+             "value": ["History is ", " because we learn about ", " and ", " that happened long ago. I can't believe people used to dress in ", " clothing and kids played with ",
+                       " and ", " instead of video games. Also, before cars were invented, people actually rode ", "! People read ", " instead of computers and tablets, and sent messages via ",
+                       " that took ", " days to arrive. I wonder how kids will view my life in ", " year(s); maybe they will ride flying cars to school and play with ", " and ", " ", "!"]
+        },
+        {
+             "title": "Star Wars",
+             "blanks": ["adjective", "noun", "adjective", "noun; place", "adjective", "adjective", "adjective", "adjective", "plural noun", "adjective", "plural noun", "plural noun", "adjective",
+                        "noun", "verb", "adjective", "verb", "plural noun; type of job", "adjective", "verb", "adjective"],
+             "value": ["Star Wars is a ", " ", " of ", " versus evil in a ", " far far away. There are ", " battles between ", " ships in ", " space and ", " duels with ", " called ",
+                       " sabers. ", " called \"droids\" are helpers and ", " to the heroes. A ", " power called The ", " ", "s people to do ", " things, like ", " ", " use The Force for the ",
+                       " side and the Sith ", " it for the ", " side."]
+        }
+]
```

Comment:
Can you look over these again, the result are a bit unsatisfactory at times. For example, there's inconsistencies with where "a" is used:
![image](https://user-images.githubusercontent.com/38372706/143785874-6a10f96d-3b81-4390-b533-01dd89784869.png)

*It says "pizzeria" but "a mouse"*